portal-api/src/v1/controllers/feedback.controller.js

```diff
@@ -38,18 +36,16 @@ exports.create = async (req, res) => {
 
   const modifiedFeedback = {
     ...req.body,
-    submittedBy: req.user.username,
-    created: now(),
+    created: getUnixTime(new Date()),
+    submittedByUserName: req.body.userDetails.username,
+    submittedByUserEmail: req.body.userDetails.email,
```

Comment:
I think an object structure would be better here

```js
submittedBy: {
  username: 'test',
  email: 'test'
}
```

Reply:
done